int32 vehicle_id
string class_name

float64 x
float64 y
float64 yaw

int32 node_id
---
bool success
string message